README.adoc

```diff
@@ -42,7 +42,7 @@ Service::
 A service is software that your component links to or depends on.
 Examples of services: MariaDB, MySQL.
 Devfile::
-A portable file responsible for your entire reproducable development environment.
+A portable file responsible for your entire reproductible development environment.
```

Comment:
`reproductible` is French I guess, it should be `reproducible`.

Reply:
Indeed 👍